ideas/for-an-inclusive-quayside/index.html

```diff
@@ -0,0 +1,111 @@
+
+                <p>It seems that there is a lack of shared design and consultation tools available to Sidewalk Labs, Waterfront Toronto, and residents of Toronto—tools that reflect the diversity and uniqueness of Toronto today, and that are open, participatory, and reciprocal. Co-design offers one potential way to help distribute the responsibility and vision for what our connected waterfront will become, and for how technologies should (and perhaps also shouldn't) be used to help make Quayside more humane, privacy-respecting, and inclusive of difference. But co-design also takes time; it requires diverse voices to be invited to the table; it needs to be tailored to the unique context of Toronto and its citizens; and it demands that all participants have equal access to the information that is essential for responsible decision-making.</p>
+
+                <p>While Sidewalk Labs have announced their partnership with OCAD University to help with accessible design, the process is still in the early stages. The University's administration signed a letter of intent with Sidewalk that expresses its interest in collaborating. Here's what it says:</p>
+
+                <blockquote>&ldquo;We are excited at the prospect of working with OCAD University. To allow us the time to evaluate a possible collaboration and, if such a collaboration is feasible, come back to you with a developed scope and terms for a possible initial project, we are suggesting entering into this letter of intent.
+
```

Comment:
I noticed that these line breaks don't appear when the text is rendered in the browser. It appears as single paragraph.

Reply:
What's the best practice for this instead of hardcoding line breaks?

Reply:
Altho this might change when we convert to markdown also...

Reply:
We can put `<p>` paragraphs inside the `<blockquote>`:

https://www.w3.org/TR/html5/grouping-content.html#the-blockquote-element

`<p>` are "flow content"

Reply:
Paragraph tags, I believe. Should be just like any other block-level container.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote#Example

Reply:
ah, @simonbates beat me to it! :smile:

Reply:
@colinbdclark @simonbates Made the changes